ComSemApp/student/views.py

```diff
@@ -1002,3 +1002,52 @@ def form_valid(self, form):
             status = 200
         
         return HttpResponse(session.id, status=status)
+
+class SpeakingPracticeReviewDetailView(StudentViewMixin, CourseViewMixin, DetailView, SpeakingPracticeInfoMixin):
```

Comment:
use `StudentCourseViewMixin` rather than `StudentViewMixin` and `CourseViewMixin`